de.wger.flutter.yml

```diff
@@ -0,0 +1,33 @@
+app-id: de.wger.flutter
```

Comment:
the app is really called "Flutter"?

Reply:
no, the app is called wger. Many moons ago we had an android app that used the id `de.wger.android`, to make the new one distinct, we simply put flutter there